client/src/helpers/hasMultiple.js

```diff
@@ -2,9 +2,11 @@
 @name hasMultiple
 @description returns true if an array contains more than one element, otherwise false
 @param {any[]} a - an array to be checked
+@param {function} filter - optional filter function to be applied to the array
 */
-export const hasMultiple = (a) => {
-  return a.length > 1
+export const hasMultiple = (a, filter) => {
+  const arr = filter ? a.filter(filter) : a
```

Comment:
Perhaps it would be useful to add the filtered item to a config file (which you may have already done in later PRs!)

Reply:
Well this helper was only used to handle presenting two computed files side by side. This wont be a feature on the portal moving forward as we will be hiding all computed files behind two dropdowns. The filtering will be done with a hook that handles user selected options to present the best fit instead of trying to display multiple computed files simultaneously.